src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -0,0 +1,328 @@
+  histogramData: Chart;
+  hits: number;
+  indexPattern: IIndexPattern;
+  minimumVisibleRows: number;
+  onAddFilter: (field: IndexPatternField | string, value: string, type: '+' | '-') => void;
+  onChangeInterval: (interval: string) => void;
+  onMoveColumn: (columns: string, newIdx: number) => void;
+  onRemoveColumn: (column: string) => void;
+  onSetColumns: (columns: string[]) => void;
+  onSkipBottomButtonClick: () => void;
+  onSort: (sort: string[][]) => void;
+  opts: {
+    savedSearch: SavedSearch;
+    config: IUiSettingsClient;
+    indexPatternList: Array<SavedObject<IndexPatternAttributes>>;
+    timefield: string;
+    sampleSize: number;
+    fixedScroll: (el: HTMLElement) => void;
+    setHeaderActionMenu: (menuMount: MountPoint | undefined) => void;
+  };
+  resetQuery: () => void;
+  resultState: string;
+  rows: Array<Record<string, unknown>>;
+  searchSource: ISearchSource;
+  setIndexPattern: (id: string) => void;
+  showSaveQuery: boolean;
+  state: AppState;
+  timefilterUpdateHandler: (ranges: { from: number; to: number }) => void;
+  timeRange?: { from: string; to: string };
+  topNavMenu: TopNavMenuData[];
+  updateQuery: (payload: { dateRange: TimeRange; query?: Query }, isUpdate?: boolean) => void;
+  updateSavedQueryId: (savedQueryId?: string) => void;
+  vis?: Vis;
+}
+
+export function DiscoverLegacy({
+  addColumn,
+  fetch,
+  fetchCounter,
+  fetchError,
+  fieldCounts,
+  histogramData,
+  hits,
+  indexPattern,
+  minimumVisibleRows,
+  onAddFilter,
+  onChangeInterval,
+  onMoveColumn,
+  onRemoveColumn,
+  onSkipBottomButtonClick,
+  onSort,
+  opts,
+  resetQuery,
+  resultState,
+  rows,
+  searchSource,
+  setIndexPattern,
+  showSaveQuery,
+  state,
+  timefilterUpdateHandler,
+  timeRange,
+  topNavMenu,
+  updateQuery,
+  updateSavedQueryId,
+  vis,
+}: DiscoverLegacyProps) {
+  const [isSidebarClosed, setIsSidebarClosed] = useState(false);
+  const { TopNavMenu } = getServices().navigation.ui;
+  const { savedSearch, indexPatternList } = opts;
+  const bucketAggConfig = vis?.data?.aggs?.aggs[1];
+  const bucketInterval =
+    bucketAggConfig && search.aggs.isDateHistogramBucketAggConfig(bucketAggConfig)
+      ? bucketAggConfig.buckets?.getInterval()
+      : undefined;
+  const [fixedScrollEl, setFixedScrollEl] = useState<HTMLElement | undefined>();
+
+  useEffect(() => (fixedScrollEl ? opts.fixedScroll(fixedScrollEl) : undefined), [
+    fixedScrollEl,
+    opts,
+  ]);
+  const fixedScrollRef = useCallback(
+    (node: HTMLElement) => {
+      if (node !== null) {
+        setFixedScrollEl(node);
+      }
+    },
+    [setFixedScrollEl]
+  );
+  const sidebarClassName = classNames({
+    closed: isSidebarClosed,
+  });
+
+  const mainSectionClassName = classNames({
+    'col-md-10': !isSidebarClosed,
+    'col-md-12': isSidebarClosed,
+  });
+
+  const sidebarCloseIcon = classNames({
+    'fa-chevron-circle-right': isSidebarClosed,
+    'fa-chevron-circle-left': !isSidebarClosed,
```

Comment:
Removing Angular should also mean removing FontAwesome wherever possible. EUI's `arrowRight` and `arrowLeft` icons are good replacements for these.

Reply:
Did this, is this ok (can't wait to have it in a nice way with EuiResizeablePanel)? 

![image](https://user-images.githubusercontent.com/463851/93302579-84326000-f7fa-11ea-9404-afc60510782e.gif)

Reply:
@andreadelrio Any feedback on this implementation of the toggle?

Reply:
@cchaos thanks for the ping. I changed the icons to `menuRight` and `menuLeft` to match what we have in Visualize and I slightly adjusted the spacing and placing of the button. Result:

![ezgif com-gif-maker](https://user-images.githubusercontent.com/4016496/93395263-6e3c9380-f82a-11ea-8754-6e090197a7f7.gif)

Reply:
@andreadelrio thanks a lot!